### ISSUE TITLE
Alternate row coloring not working correctly when results/timetable page is filtered

### ISSUE THREAD

**Issue by Breit** [Breit/DBRegatta#6]
If you click on one particular team on the timetable or results page to filter the races for that particular team, the alternating row coloring is not working as expected.

Fixes in Javascript necessary: `.detach()` hidden rows from DOM instead of just mark them as `display: none`!
